config/coreboot-t440p.config

```diff
@@ -0,0 +1,18 @@
+# CONFIG_USE_BLOBS is not set
+CONFIG_VENDOR_LENOVO=y
+CONFIG_NO_POST=y
+CONFIG_CBFS_SIZE=0x800000
+CONFIG_IFD_BIN_PATH="@BLOB_DIR@/t440p/ifd.bin"
+CONFIG_ME_BIN_PATH="@BLOB_DIR@/t440p/me.bin"
+CONFIG_GBE_BIN_PATH="@BLOB_DIR@/t440p/gbe.bin"
+CONFIG_HAVE_IFD_BIN=y
+CONFIG_BOARD_LENOVO_THINKPAD_T440P=y
+CONFIG_LINUX_COMMAND_LINE="intel_iommu=igfx_off"
```

Comment:
Exactly. Otherwise the iommu is configured incorrectly since first kernel boot would try to isolate Intel iommu and the kexec'ed kernel would not be able set differently.

Coreboot defined kernel option defines Heads behavior.